docs/schemas/v1/definitions.json

```diff
@@ -344,6 +344,21 @@
                 }
               ],
               "description": "Display mode of the page. This can also be set at runtime with the toolpad-display query parameter"
+            },
+            "layout": {
```

Comment:
The team had some discussion on this - instead of `layout`, could we have.`container` page property with 2 possible values for now: `fluid` and `none`? 
`none` would be the full-width unrestricted page you proposed in this PR, and `fluid` would be the previous behavior. We feel that this would align closer to the terminology of the MUI `Container` component https://mui.com/material-ui/react-container/